addons/account/security/ir.model.access.csv

```diff
@@ -22,6 +22,7 @@ access_account_fiscal_position_account,account.fiscal.position all,model_account
 
 access_product_product_account_user,product.product.account.user,product.model_product_product,group_account_readonly,1,0,0,0
 access_product_product_account_manager,product.product.account.manager,product.model_product_product,account.group_account_manager,1,1,1,1
+access_product_template_account_manager,product.template.account.manager,product.model_product_template,account.group_account_manager,1,1,1,1
```

Comment:
Don't we need it for `account.group_account_readonly` as well?

Reply:
As these users are only supposed to read accounting documents and should not be able to modify them or create new ones,I think they don't need any access to products.